common/src/main/resources/schema/development/schema.json

```diff
@@ -167,17 +170,123 @@
         }
       }
     },
-    "storageObject": {
+    "azureStorageObject": {
       "type": "object",
       "properties": {
         "region": {
-          "enum": ["southamerica-west1", "us-central1", "us-east1", "us-east4", "us-west1", "us-west4", "europe-north1", "europe-west1", "europe-west4", "asia-east1", "asia-southeast1"]
+          "enum": [
+            "eastus",
+            "eastus2",
+            "southcentralus",
```

Comment:
It will be interesting to see how this (having a restrictive list of regions) works as the list of region changes over time. I also wonder how much value users will get from being able to do per-region filtering or searching. On the other hand it's good to have data match up across datasets and if we know the list of possible values, why not check for them.

Reply:
Yeah, I am most worried about data matching up between datasets. I also could see a case for filtering by region, if a research project has restrictions that data used be hosted in a particular place. (Although more often I suspect that would matter on the custodian side)